motes.md

```diff
@@ -0,0 +1,42 @@
+* `BLS_WITHDRAWAL_PREFIX`
+  * Why int rather than bytes?
+* `MIN_SEED_LOOKAHEAD`
+  * Is this actually tunable?
+  * If so, what are the reprecussions?
+* `ACTIVATION_EXIT_DELAY`
+  * Reaquaint with purpose
+* AttesterSlashings
+  * `MAX_ATTESTER_SLASHINGS` is 1.
+  * Are there scenarios in which validators can create more effective slashable
+    messages than can be included on chain? For example, Validators split up to
+    create double attestations for checkpoints but different (junk) crosslink
+    data to prevent them from being aggregatable to the fullest
+  * Max is for block size, no?
+* Signature domains
+  * Max 4byte ints
+* `Version` not defined in one of the lists of custom types (2!!). ensure in spec
+* `PendingAttestation`
+  * Don't think `proposer_index` is actually necessary here because effective
+    balance is stable until end of epoch so can do dynamic lookups
+* is_genesis_trigger
+  * only run at ends of blocks to preserve invariant that eth1data.deposit_root
+    is the deposit root at the _end_ of an eth1 block
+* `Attestation` 
+  * why bitfields not together?
+* `Transfer`
+  * replay mechanism... say the slot gets missed and you sign another transfer
+  * in a fork you could include both transfers
```

Comment:
I discussed this with Vitalik at Devcon. The current replay mechanism is quite bad and user unfriendly. The latest suggestion is to have a buffer of the latest `n` (where `n` is sufficiently large) transfer roots, and use that as replay protection.

Since we know we want to change the current transfer logic, can we please remove transfers from phase 0? Including the current logic is now actively misleading.